File: uri/python/1036.py
# -*- coding: utf-8 -*-
import math

a, b, c = input().split(" ")

a = float(a)
b = float(b)
c = float(c)

delta = 0
raiz1 = 0
raiz2 = 0

delta = (b * b) - (4 * a * c)

if a <= 0:
    print('Impossivel calcular')
elif delta < 0:
    print('Impossivel calcular')
elif delta >= 0 and a > 0:
    raiz1 = (-b + math.pow(delta, 1/2)) / (2 * a)
    raiz2 = (-b - math.pow(delta, 1/2)) / (2 * a)
    print('R1 = {:.5f}'.format(raiz1))
    print('R2 = {:.5f}'.format(raiz2))

File: uri/C/beginner/1036.c
#include <stdio.h>
#include <math.h>

int main ()
{

	double A, B, C, delta, raiz1, raiz2;

	scanf("%lf %lf %lf", &A, &B, &C);

	delta = pow(B, 2) - 4 * A * C;

	if(delta >= 0 && A != 0){
		raiz1 = (-B + sqrt(delta))/ (2 * A);
		raiz2 = (-B - sqrt(delta))/ (2 * A);
		printf("R1 = %.5lf\n", raiz1);
		printf("R2 = %.5lf\n", raiz2);
	}

		
	if(delta < 0 && A == 0 || delta < 0 && A != 0 || delta > 0 && A ==0){

		printf("Impossivel calcular\n");
	}

	return 0;
}
